src/Http/Http.Extensions/test/RequestDelegateGenerator/CompileTimeCreationTests.cs

```diff
@@ -672,6 +672,7 @@ public static IEndpointRouteBuilder MapPost(this IEndpointRouteBuilder app, int
     public async Task TestHandlingOfGenericWithNullableReferenceTypes()
     {
         var source = """
+#nullable enable
```

Comment:
Is this the only test that tests nullability in user code 😮
Kind of surprising.

Reply:
@BrennanConroy The other tests do use nullable annotations but they make use of the source template that is in the base class which enables it there:

https://github.com/dotnet/aspnetcore/blob/2c0ee4c870556d823ea5a0b35a8ba1b8ae81142c/src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateCreationTestBase.cs#L268